priv/repo/migrations/20171001195406_add_summary_to_offers.exs

```diff
@@ -0,0 +1,10 @@
+defmodule ElixirJobs.Repo.Migrations.AddSummaryToOffers do
+  use Ecto.Migration
+
+  def change do
+    alter table(:offers) do
+      add :summary, :string
```

Comment:
We should set maximum size in this field too! That will avoid an extra layer of security against content attacks